service-api/module/Application/src/Controller/YotiController.php

```diff
@@ -72,12 +73,23 @@ public function getSessionStatusAction(): JsonModel
             $this->getResponse()->setStatusCode(Response::STATUS_CODE_400);
             return new JsonModel(['error' => 'Missing uuid']);
         }
-        //@TODO look up actual sessionId from case and case where this is not created
-        $sessionId = 'AJDAHDFSH';
-        $session = $this->yotiService->retrieveResults($sessionId);
+
+        $caseData = $this->dataQuery->getCaseByUUID($uuid);
+        if (! $caseData) {
+            $this->getResponse()->setStatusCode(Response::STATUS_CODE_400);
+            return new JsonModel(['error' => 'Case not found']);
+        }
+
+        $sessionId = $caseData->yotiSessionId;
+        if ($sessionId === '00000000-0000-0000-0000-000000000000') {
```

Comment:
I'm curious why this particular check exists.

Reply:
Due to dynamodb quarks, we need a value for an indexed attribute on record creation, so need to check against default value as a false positive